scripts/start/prepare_and_run.R

```diff
@@ -511,25 +511,29 @@ prepare <- function() {
   # Merge GAMS files
   message("\nCreating full.gms")
 
-  # only compile the GAMS file to catch compilation errors and create a dump file with the full code
+  # only compile the GAMS file to catch compilation errors and create a dump
+  # file with the full code
   modelFilePathStem <- substr(tmpModelFile, 1, nchar(tmpModelFile) - 4)
   dumpFilePath <- paste0(modelFilePathStem, ".dmp")
   listFilePath <- paste0(modelFilePathStem, ".lst")
   logFilePath <- paste0(modelFilePathStem, ".log")
 
-  exitcode <- system2(cfg$gamsv, c(tmpModelFile, "action=c", "dumpopt=21", "logoption=", cfg$logoption))
-  if ( 0 < exitcode ) {
-    stop(paste("Compiling", tmpModelFile, "failed, stopping. Check",
-               logFilePath, ",", dumpFilePath, ", and", listFilePath, "for details." ))
-  }
-
+  exitcode <- system2(cfg$gamsv, c(tmpModelFile, "action=c", "dumpopt=21",
+                                   "logoption=", cfg$logoption))
   file.rename(dumpFilePath, file.path(cfg$results_folder, "full.gms"))
   file.rename(listFilePath, file.path(cfg$results_folder, "main.lst"))
   file.rename(tmpModelFile, file.path(cfg$results_folder, "main.gms"))
```

Comment:
Is it possible to break gams so badly that the `dmp` and/or `lst` files won't be produced? If so, we have to guard those with a `file.exists` as well.

Reply:
> Is it possible to break gams so badly that the `dmp` and/or `lst` files won't be produced? 

Probably.  David once managed to make GAMS spit out a Fortran error.
Everything is possible …